rtsf-at-checkout-device-scale/Dockerfile

```
@@ -12,21 +12,22 @@ LABEL license='SPDX-License-Identifier: Apache-2.0' \
 RUN sed -e 's/dl-cdn[.]alpinelinux.org/nl.alpinelinux.org/g' -i~ /etc/apk/repositories
 
 # add git for go modules
-RUN apk update && apk add make git
+RUN apk update && apk add --no-cache make git gcc libc-dev zeromq-dev linux-headers
 
 COPY go.mod .
 
 RUN go mod download
 
 COPY . .
 
-RUN CGO_ENABLED=0 go build -o device-scale cmd/main.go
+RUN go mod tidy
+RUN CGO_ENABLED=1 GOOS=linux go build -o device-scale -ldflags='-s -w' -a -installsuffix cgo cmd/main.go
```

Why adding `ldflags='-s -w'` and `GOOS=linux` ? 
I should be able to build and run the binary on Windows

```suggestion
go build -tags no_zmq -o device-scale cmd/main.go
```